frontend/src/services/volunteer/volunteer.api.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from 'react-query';
+import { OrderDirection } from '../../common/enums/order-direction.enum';
+import { getAccessCodes } from './volunteer.service';
+
+export const useGetAccessCodesQuery = (
```

Comment:
Why is this in a file called volunteer.api.ts?

Reply:
Aren't access codes related to volunteers? In the side menu they are under volunteers, so i assumed that's the .api they would find themselves in.